### PR TITLE
View

### PR DESCRIPTION
**Pull request by Olamarx**
## Implemented the view functionality

In this milestone, I:
 - did not style the project because it is not a core requirement
 - Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/posts/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <% (1..limit).each do |the_page| %>
+      <li><%= link_to the_page, user_posts_path(@user, page: the_page) %></li>
+    <% end %>
+    <li>
+      <%= link_to "Next", user_posts_path(@user, page: next_page) %>
+    </li>
+  </ul>
+</nav>
+
+</section>
```
Comment (tresorsawasawa) [inline app/views/posts/_user.html.erb]: - [ ] As required for the project, kindly make sure your design matches to the expected **[wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=ABIFLA3WTIGWG4Z5XBP2BQLBGXZKY)**

File: app/views/users/_showPosts.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+</p>
+<p>
+<small>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></small>
+</p>
+<% end %>
+<button type="button">
+    <%= link_to "See all posts", user_posts_path(@ind_user)%>
+  </button>
+<% end %>
+</section>
```
Comment (tresorsawasawa) [inline app/views/users/_showPosts.html.erb]: - [ ] Kindly make sure your design sticks as much as possible to the **[wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)** as required for the project.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Olamarx 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Olamarx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**